docs/overview/learning-path-transition.md

```diff
@@ -37,16 +37,9 @@ There are a lot of links in those articles, but if you're transitioning to Offic
 
 ## Step 2: Install tools and create your first add-in
 
-You've got the big picture now, so dive in with one of our quick starts. For purposes of learning the platform, we recommend the Excel quick start. There's a version based on Visual Studio and another based on Node.js and Visual Studio Code. If you're transitioning from VSTO add-ins, you'll probably find the Visual Studio version easier to work with.
+You've got the big picture now, so dive in with one of our quick starts. For purposes of learning the platform, we recommend the [Excel quick start for Visual Studio](../quickstarts/excel-quickstart-vs.md).
 
-- [Visual Studio](../quickstarts/excel-quickstart-jquery.md?tabs=visualstudio)
-- [Node.js and Visual Studio Code](../quickstarts/excel-quickstart-jquery.md?tabs=yeomangenerator)
-
-## Step 3: Code
-
-You can't learn to drive by reading the owner's manual, so start coding with this [Excel tutorial](../tutorials/excel-tutorial.md). You'll be using the Office JavaScript library and some XML in the add-in's manifest. There's no need to memorize anything, because you'll be getting more background about both in a later step.
```

Comment:
Why are we removing this step? Back when this article was created, this step was included because our feedback was that newbie add-in devs want to start coding early in the learning process.

Reply:
I removed this because it links to a Yo Office based tutorial. I could change the recommendation from Excel to PowerPoint, since there is a VS version of the long-form tutorial. I don't there are other VS-based walkthroughs right now.

Reply:
> I removed this because it links to a Yo Office based tutorial. I could change the recommendation from Excel to PowerPoint, since there is a VS version of the long-form tutorial. I don't there are other VS-based walkthroughs right now.

If there's a way to keep this step, I think we should keep it. The structure of this article was pretty thoroughly worked out by Reeza, Michelle, and I in light of both internal and external feedback.

Reply:
@Rick-Kirkham I reworked that section back in. How do you feel about this iteration? Is this a good compromise?